modules/exadsBidAdapter.md

```diff
@@ -0,0 +1,362 @@
+* The tcString is the iabtcf consent string for gdpr;
+* Uncomment the cache instruction if you need to configure a cache server (e.g. for instream video)
+
+```
+pbjs.setConfig({
+    debug: false,
+    //cache: { url: "https://prebid.adnxs.com/pbc/v1/cache" },
+    consentManagement: {
+        gdpr: {
+            cmpApi: 'static',
+            timeout: 1000000,
+            defaultGdprScope: true,
+            consentData: {
+                getTCData: {
+                    tcString: consentString,
+                    gdprApplies: false // set to true to pass the gdpr consent string
+                }
+            }
+        }
+    }
+});
+
+```
+
+# Test Parameters
+ 
+Now you will find the different parameters to set, based on publisher website. They are optional unless otherwise specified.
+ 
+#### RTB Banner 2.4
+
+* **zoneId** (required) - you can get it from the endpoint created after configuring the zones (integer)
+* **fid** (required) - you can get it from the endpoint created after configuring the zones (string)
+* **partner** (required) - currently we support rtb 2.4 ("rtb_2_4") only (string)
+* **siteId** (recommended) - unique Site ID (string)
+* **banner.sizes** (required) - one integer array - [width, height]
+* **catIab** - IAB category ID (string array)
+* **userIp** (required) - IP address of the user, ipv4 or ipv6 (string)
+* **userId** (*required) - unique user ID (string). *If you cannot generate a user ID, you can leave it empty (""). The request will get a response as long as "user" object is included in the request
+* **country** - country ISO3
+* **impressionId** (required) - unique impression ID within this bid request (string)
+* **keywords** - keywords can be used to ensure ad zones get the right type of advertising. Keywords should be a string of comma-separated words
+* **bidfloor** - minimum bid for this impression (CPM) / click (CPC) and account currency (float) 
+* **bidfloorcur** - currency for minimum bid value specified using ISO-4217 alpha codes (string)
+* **mimes** - list of supported mime types. We support: image/jpeg, image/jpg, image/png, image/png, image/gif, image/webp, video/mp4 (string array)
+* **endpoint** (required) - EXADS endpoint (URL)
+
+##### RTB Banner 2.4 (Image)
+* **image_output** - indicates output format for image banners (string)
+    * options are **xml** or **html**. Default value is **xml** in case field is not provided
+
+```javascript
+
+adUnits = 
+    [{  code: 'postbid_iframe', // the frame where to render the creative
+        mediaTypes: {
+            banner: {
+                sizes: [300, 250]
+            }
+        },
+        bids: [{
+            bidder: 'exadsadserver',
+            params: {
+                    zoneId: 12345,
+                    fid: '829a896f011475d50da0d82cfdd1af8d9cdb07ff',
+                    partner: 'rtb_2_4',
+                    siteId: '123',
+                    catIab: ['IAB17-15'],
```

Comment:
this is not yet supported, so we should probably remove it

Reply:
Do you mean catIab?

Reply:
removed catIab